Java2_HW_5/src/main/java/HomeWork5.java

```diff
@@ -0,0 +1,94 @@
+public class HomeWork5 {
+
+    public static void main(String[] args) {
+
+        final int size = 10000000;
+        float[] array = new float[size];
+
+    // Первый метод просто бежит по массиву и вычисляет значения.
+        fillArrayOnes(array);
+        CulcArray1(array);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.